apps/condo/pages/ticket/index.tsx

```diff
@@ -1075,18 +1091,18 @@ const TicketsPage: ITicketIndexPage = () => {
                                                 }
                                             />
                                         </Col>
+                                        {
```

Comment:
Why not to use `Space` component and switch its direction instead of slicing-and-dicing the structure?